Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -69,11 +69,17 @@ public class RocksDBStore : BaseBlockStatesStore
         /// <param name="blockCacheSize">The capacity of the block cache.</param>
         /// <param name="txCacheSize">The capacity of the transaction cache.</param>
         /// <param name="statesCacheSize">The capacity of the states cache.</param>
+        /// <param name="maxTotalWalSize">The number to configure max_total_wal_size RocksDB option.
+        /// </param>
+        /// <param name="keepLogFileNum">The number to configure keep_log_file_num RocksDB option.
+        /// </param>
         public RocksDBStore(
             string path,
             int blockCacheSize = 512,
             int txCacheSize = 1024,
-            int statesCacheSize = 10000
+            int statesCacheSize = 10000,
+            ulong? maxTotalWalSize = null,
```

Comment:
I guess we should use `long` instead of `ulong` on public interface to keep CLS compatibility.

Reply:
I used its type as `ulong` because the type of `DbOptions.SetKeepLogFileNum()`'s argument was `ulong`. I will amend it to keep CLS compatibility.

Reply:
I guess explicit cast(`(ulong)`) will work...
```csharp
long maxTotalWalSize = 0L;
 _options.SetMaxTotalWalSize((ulong)maxTotalWalSize);
```

Reply:
I'm not sure it is okay to use `ulong` type for convert to keep CLS compatibility. I guess the `ulong` shouldn't appear anywhere.

Reply:
As far I know, CLS compliance apply only to public interface, not to its private implementation. (see also: https://docs.microsoft.com/en-US/dotnet/standard/language-independence-and-language-independent-components#Rules)

Reply:
How about giving up CLS compatibility only for Libplanet.RocksDBStore project?  Although we can cast `long` to `ulong`, we still lost its upper bits (9,223,372,036,854,775,808 to 18,446,744,073,709,551,615).

Reply:
If we need higher value for these options, I agree with giving up.

Reply:
Actually, I checked the RocksDB C++ core also used the option's type as `uint64` (`ulong`). So I will give up it and stay with this code. @longfin